internal/service/implement/transaction_service.go

```diff
@@ -204,7 +225,53 @@ func (service *TransactionService) InternalTransfer(ctx *gin.Context, transferRe
 		return nil, err
 	}
 
-	// notify, response history
+	// send notification to ws
+	notificationForSourceCustomerResp := &model.NotificationResponse{
+		DeviceId:      int(sourceCustomer.Id),
+		Name:          sourceCustomer.Name,
+		Amount:        int(existsTransaction.Amount),
+		TransactionId: existsTransaction.Id,
+		Type:          "outgoing_transfer",
+		CreatedAt:     existsTransaction.CreatedAt,
+	}
+
+	notificationForTargetCustomerResp := &model.NotificationResponse{
+		DeviceId:      int(targetCustomer.Id),
+		Name:          targetCustomer.Name,
+		Amount:        int(existsTransaction.Amount),
+		TransactionId: existsTransaction.Id,
+		Type:          "incoming_transfer",
+		CreatedAt:     existsTransaction.CreatedAt,
+	}
+
+	// // notify, response history
+	sourceContent, _ := json.Marshal(notificationForSourceCustomerResp)
+	notificationForSourceCustomer := &entity.Notification{
+		Type:    notificationForSourceCustomerResp.Type,
+		Content: string(sourceContent),
+		IsSeen:  false,
+		UserID:  sourceCustomer.Id,
+	}
+	err = service.notificationRepository.CreateCommand(ctx, notificationForSourceCustomer)
```

Comment:
nên save trong notificationClient

Reply:
Lý do: ko nên để logic send noti ở đây, tự nhiên cái noti lỗi là ngưng flow chuyển tiền , trong khi cái noti nó không quan trọng trong cái flow chuyển tiền này?

Reply:
already move CreateCommand to notificationClient